.github/workflows/autoblocks-testing.yml

```diff
@@ -32,6 +32,9 @@ jobs:
       - name: Install dependencies
         run: poetry install
 
+      - name: Run mypy
```

Comment:
would prefer to add this to pre-commit but couldn't figure out how to set it up properly in a monorepo. i was running into the same issue as https://github.com/pre-commit/pre-commit/issues/2951 but the maintainer closed it lol (and i could not find the dupes he was referring to)